posts/2020-09-25-signed-certificate-with-acme-200010.adoc

```diff
@@ -1,12 +1,12 @@
 ---
 layout: post
-title: "Install a signed certificate with ACME Certificate Authority with acmeCA-2.0"
+title: "Install a signed certificate with ACME Certificate Authority with acmeCA-2.0 and significant bug fixes on Open Liberty 20.0.0.10"
```

Comment:
- remove "and significant bug fixes" - not necessary or exciting in the title

Reply:
Done